src/dbsync/testtool/cmdArgsHelper.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Wazuh DBSYNC
+ * Copyright (C) 2015-2020, Wazuh Inc.
+ * July 02, 2020.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _CMD_LINE_ARGS_HELPER_H_
+#define _CMD_LINE_ARGS_HELPER_H_
+
+#include <string>
+#include <sstream>
+#include <vector>
+#include <iostream>
+
+class CmdLineArgs
+{
+public:
+    CmdLineArgs(const int argc, const char* argv[])
+    {
+        m_cmdLineArgs.reserve(argc);
+        for (int i = 1; i < argc; ++i)
```

Comment:
in here you can check if the number of parameters is the expected one and parse them.